packages/button/src/ClearButton.ts

```diff
@@ -66,9 +66,16 @@ export class ClearButton extends SizedMixin(StyledButton, {
 
     /**
      * The visual variant to apply to this button.
+     * @deprecated Use `static-color='white'` instead.
      */
     @property({ reflect: true })
-    public variant: 'overBackground' | '' = '';
+    public variant: 'overBackground' | undefined;
```

Comment:
@caseyisonit If we're okay with this feature, all the VRTs are passing on this. Are there tests or test cases I can write up to validate the staticColor? Can you help me with how I might surface the deprecation notice to consumers?